src/stories/Jobs/CanUpdateJob/endpoint.spec.js

```diff
@@ -87,32 +48,13 @@ describe("Test API Jobs/CanUpdateJob", () => {
     expect(respondResult.statusCode).toBe(200);
     expect(respondResult.json()).toMatchObject({
       uuid: expect.any(String),
-      user_uuid: contextClassRef.user.user_uuid,
-      company_name: "Betalectic Pvt Lts",
-      title: "Software Engineer",
-      experience: "1 year",
-      location: {
-        country: "India",
-        city: "Hyderabad",
-      },
-      description: "",
-      requirements_attachments: [],
-      top_skills: ["React JS", "Node JS"],
-      employment_type: "private",
-      salary_offer_band: {
-        currency: "INR",
-        range: {
-          from: "500000",
-          to: "600000",
-        },
+      recruiter_uuid: "001",
+      title: "React JS",
+      job_description: {
+        comapny: "Facebook",
+        experience:"3yrs"
```

Comment:
can be a bit confusing. But its fine for now.

Reply:
ok